Berksfile

```diff
@@ -2,4 +2,4 @@ source 'https://supermarket.chef.io'
 
 cookbook 'build-essential', '~> 2.1.3'
 cookbook 'homebrew', '~> 1.9'
-cookbook 'pantry'
+cookbook 'pantry', path: '../pantry/'
```

Comment:
This shouldn't have the path specified. Consumers of this repo won't have the pantry cookbook yet.